src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -249,7 +249,7 @@ protected Task Stop()
             string helpParams,
             RequestContext<object> requestContext)
         {
-            if (helpParams == null) { helpParams = "Get-Help"; }
+            if (helpParams == "") { helpParams = "Get-Help"; }
```

Comment:
Might be worth just making this `string.IsNullOrEmpty(helpParams)`. We lose nothing really if the string is never `null`

Reply:
I don't know what you're talking about... it's definitely string.IsNullOrEmpty *whistles innocently*